packages/dashboard/src/components/rmf-dashboard.tsx

```diff
@@ -153,12 +154,25 @@ export function RmfDashboard(props: RmfDashboardProps) {
   );
 
   const [userProfile, setUserProfile] = React.useState<UserProfile | null>(null);
+  const [authReady, setAuthReady] = React.useState(false);
```

Comment:
This remains false after a page refresh, and doesn't get updated so nothing gets rendered. Could you help look into it?

Reply:
All the branches from the below effect ends with setting it to `true`, the only way I see for it to be false is that it is not finishing the promises, which indicate some timeout issues with kc or api server.

Reply:
thanks for confirming, the issue was a KC timeout